engine/inc/info/info.hpp

```diff
@@ -15,6 +15,10 @@
 #include "time/timer.hpp"
 #include "./version.hpp"
 
+#define LOGGING_STDOUT (0)

```

Comment:
> Please replace this C-Style code to C++ enums (like [here](https://github.com/h4570/tyra/blob/master/engine/inc/renderer/core/3d/bbox/core_bbox_frustum.hpp)).
> 
> `_EESIO` can be misleading for Tyra beginners, what do you think about `LOGGING_UART`?

Maybe. If we explicitly say this is for EE.
Because DECKARD slims also have an additional UART for their emulated IOP

Rn I don't have my PC at hand. Will fix later